vibi-dpu/src/utils/aliases.rs

```diff
@@ -0,0 +1,27 @@
+use crate::{core::utils::get_aliases, db::aliases::get_handles_from_db};
+
+use super::review::Review;
+
+
+pub async fn get_login_handles(git_alias: &str, review: &Review) -> Option<Vec<String>> {
+    // Get aliases from db
+    let handles_db_opt = get_handles_from_db(git_alias, &review.provider());
+    if handles_db_opt.is_none() {
+        let server_aliases_opt = get_aliases(review).await;
```

Comment:
can we please have a better name here for the function? `get_handles_from_server`?

Reply:
done